### DISCUSSION TITLE
How can i acces a svelte store in a sveltekit endpoint?

### DISCUSSION THREAD

**Discussion by tryggs** [sveltejs/kit#5878] [Q&A] [answered]
I believe that my title sums my question up pretty good: )

thanks,

**Comment by j4w8n** [comment, marked as answer]
If endpoint means a standalone endpoint, then the you can't import stores. Best to pass in data to a standalone endpoint via `event.locals`. See [handle](https://kit.svelte.dev/docs/hooks#handle)

If endpoint means a page endpoint, then import the store, either from the .js file you created it in or directly from SvelteKit if it's one they provide. For custom stores, see [this](). If SvelteKit, see [$app/stores](https://kit.svelte.dev/docs/modules#$app-stores)
